### PR TITLE
Deleting employee_stuff package - RIP :(

### PR DESCRIPTION
**Pull request by pwright08**
plz review guys

### CODE REVIEW

**Comment by AdamKane44** [comment]
4/5 stars - needs more deletion

File: src/employee_stuff/Employee.java [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-    private String lname;
-    private float salary;
-    private String address;
-    private String postcode;
-    private String phone_no;
-    private Date dob;
-    private String nin;
-    private String email;
-
-    public Employee(int newID, String newFName, String newLName, float newSalary,
```
Comment (ElliotB-96) [inline src/employee_stuff/Employee.java]: Constructor should offer overloaded versions

File: src/employee_stuff/EmployeeController.java [hunk 2]
```diff
@@ -1,60 +0,0 @@
...
-import java.util.ArrayList;
-import java.sql.ResultSet;
-import java.util.Date;
-
-
-public class EmployeeController {
-    static DatabaseConnector dc;
-
-    public static void connectToDB() {
-        dc = DatabaseConnector.initConnection("root", "Nylander62", "localhost");
```
Comment (AaronMKainos) [inline src/employee_stuff/EmployeeController.java]: Magic strings and plaintext password.

Comment (spdevlin) [inline src/employee_stuff/EmployeeController.java]: SAFETY!

File: src/employee_stuff/ProjectController.java [hunk 3]
```diff
@@ -1,78 +0,0 @@
...
-    private static final Logger log= Logger.getLogger( ProjectController.class.getName() );
-
-    public static void connectToDB() {
-        dc = DatabaseConnector.initConnection("application", "password", "localhost");
-    }
-
-    public static ArrayList<Project> getAllProjects() {
-        ArrayList<Project> projects = new ArrayList<>();
-        ResultSet rs;
-        connectToDB();
```
Comment (spdevlin) [inline src/employee_stuff/ProjectController.java]: Reconnecting to db for each call

**Pull request closed without merging**